tests/manager/manager-set-uri-options-002.phpt

```diff
@@ -50,7 +50,7 @@ Deprecated: MongoDB\Driver\Manager::__construct(): The "context" driver option i
 
 Deprecated: MongoDB\Driver\Manager::__construct(): The "allow_self_signed" context driver option is deprecated. Please use the "tlsAllowInvalidCertificates" URI option instead.%s
 OK: Got MongoDB\Driver\Exception\ConnectionTimeoutException
-No suitable servers found (`serverSelectionTryOnce` set): [%scalling ismaster on '%s']
+No suitable servers found (`serverSelectionTryOnce` set): [%scalling %s on '%s']
```

Comment:
I realize this was in the older diff, but is there any reason this is `%scalling` instead of `%s calling`? Might be a good chance to change it.

Reply:
Ah, I missed that. Will change that as well.